cmd/honeycloudfront/main.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/honeycombio/honeyelb/logbucket"
+	"github.com/honeycombio/honeyelb/options"
+	"github.com/honeycombio/honeyelb/publisher"
+	"github.com/honeycombio/honeyelb/state"
+	libhoney "github.com/honeycombio/libhoney-go"
+	flag "github.com/jessevdk/go-flags"
+)
+
+var (
+	opt        = &options.Options{}
+	BuildID    string
+	versionStr string
+)
+
+func init() {
+	// set the version string to our desired format
+	if BuildID == "" {
+		versionStr = "dev"
+	} else {
+		versionStr = "1." + BuildID
```

Comment:
the build number already has a 1. at its beginning. Are you intentionally making it 1.1.X instead to get the semver semantics? If so, let's change the travis file to pass in 1.1.x instead of chaining it together in two places.

Reply:
Hm, it's just nicked from other code I think. Based on some of the user agents I've seen it clearly seems to be in error. I'll remove it.